server/src/internalClusterTest/java/org/opensearch/search/nested/SimpleNestedExplainIT.java

```diff
@@ -30,6 +30,11 @@
  */
 public class SimpleNestedExplainIT extends OpenSearchIntegTestCase {
 
+    @Override
+    protected int numberOfShards() {
+        return 1;
+    }
+
```

Comment:
I actually wonder if this change by itself is enough to stabilize the tests. We can still use `indexRandomForMultipleSlices("test")` but hopefully the docs won't get spread across multiple shards in that case then.

Reply:
Yeah, 👍 to this

Reply:
this change should be enough to stabilize the tests. We dont need to call the `indexRandomForMultipleSlices` as the sole purpose of that function is to create multiple slice which is happening by ingesting two docs.

Reply:
also there is no deletion of docs done here so it should stabilize the flakiness of the range seen in the tests.

Reply:
ah, that's right, nonetheless method is called `indexXxx`, there are deletes as well